src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
+    this.globalConfig = {};
+    const start = +new Date();
+    return new Promise((resolve) => {
```

Comment:
if you wanted to get rid of the extra boilerplate around wrapping things in a promise, you can explore using `utils.promisify`: http://2ality.com/2017/05/util-promisify.html

Reply:
I tried running it with util.promisify but flow crashes in this situation. (I also gave a try to run it with execSync, as this command is blocking me, but I got the same flow error) I think it worth investigation for future version.